docs/instrumentation-list.yaml

```diff
@@ -605,11 +671,14 @@ libraries:
       - org.apache.shenyu:shenyu-web:[2.4.0,)
     configurations:
     - name: otel.instrumentation.apache-shenyu.experimental-span-attributes
-      description: Enables experimental span attributes for Apache Shenyu instrumentation.
+      description: |
+        Enables experimental `apache-shenyu.meta.` prefixed span attributes like `app-name`, `service-name`, `context-path`, `param-types`, `id`, `method-name`, `rpc-type`, `path`, and `rpc-ext`.
```

Comment:
If we are going to list all the attributes then I think we can remove the word `like`. Also there is a comma before the last and, is that ok?
Are we listing the experimental attributes for other instrumentations too? If we wish to do that then it might be better to also provide descriptions for these attributes.

Reply:
i was on the fence about this one, primarily because there are so many. I've added some similar descriptions for other instrumentations when there are only 1 or 2 attributes added, but not consistently. ([example 1](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/instrumentation-list.yaml#L1024), [example 2](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/instrumentation-list.yaml#L2209)). I opened #14247 to come up with a new way to display this type of thing, but it's more geared towards a specific type of instrumentation.  For now I think I'll just try and be more consistent in terms of adding these descriptions. Do you have any thoughts or opinions?